### PR TITLE
shop database project content updated

### PR DESCRIPTION
**Pull request by babalwa-mbolekwa**
Related issues: [please specify]
Added content for the Shop database project to specify how the structure of the project repository should look like.

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Comment by Sbonelo01** [comment]
Hey @Babalwa01 , Is this PR linked to a specific issue? otherwise, I'm not sure what criteria I should review it against

**Comment by babalwa-mbolekwa** [comment]
> Hey @Babalwa01 , Is this PR linked to a specific issue? otherwise, I'm not sure what criteria I should review it against

Hey @Sbonelo01 There isn't an issue yet. I have come across many instances where recruits put all the Sql commands in one file, name it _shopDatabase.sql_ and some of the repos don't have a _.yml_ file. I have based it on the reviews I received from you earlier when I did my shop database project

File: content/sql/shop-project/_index.md [hunk 1]
```diff
@@ -22,6 +22,13 @@ title: Shop Database using sql
 - Instead of installing Postgres on your computer, you can launch it with a docker composition
 - MySQL is nice and lot's of people use it in industry, but it doesn't actually implement standard SQL, it sort of does it's own thing a bit. Postgres is a much more standard DB, and industry loooooves it
 
+## Structure:
+
+- Your repository should have a **.yml** file 
+- Your repository should have **.sql** files
+- The different SQL commands should be saved in different descriptive script files i.e.
+when the reviewer is looking for the commands you used for creating the database they should be able to navigate to a file named **create-database.sql**  , navigate to a file named **create-tables.sql** for the commands used to create the database tables , and navigate to the files that contain the commands used for inserting table records and querying the database
```
Comment (Sbonelo01) [inline content/sql/shop-project/_index.md]: This is good to point out. Can you please make this (the points mentioned in the sentence starting from line 30) in bullet or list form? Just so it is clearer.

**Review by Sbonelo01 — CHANGES_REQUESTED** [review]

**Review by Sbonelo01 — APPROVED** [review]
LGTM

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**